### DISCUSSION TITLE
What is Unsigned ?

### DISCUSSION THREAD

**Discussion by UsefullProgrammer** [EDCD/EDMarketConnector#2294] [General]
EDMarketConnector_Installer_Unsigned_5.12.0.exe what is Unsigned?

**Comment by Rixxan** [comment]
Hey there!

The signed vs unsigned releases are explained here:
https://github.com/EDCD/EDMarketConnector/wiki/Code-Signing-and-EDMC

Unsigned in this instance means that this is the raw, GitHub built
installer and did not receive a cryptographic signing signature prior to
release/distribution.
